README.md

```diff
@@ -153,6 +153,12 @@ You can configure the server with the `aiida_optimade/config.json` file or set c
 
 To learn more about this, see the [`optimade-python-tools`](https://github.com/Materials-Consortia/optimade-python-tools) repository.
 
+### Using AiiDA group for curated data
+
+An [AiiDA Group](https://aiida.readthedocs.io/projects/aiida-core/en/latest/howto/data.html#organizing-data) can be used to curate data and server only this curated data through the OPTIMADE server.
```

Comment:
```suggestion
An [AiiDA Group](https://aiida.readthedocs.io/projects/aiida-core/en/latest/howto/data.html#organizing-data) can be used to curate data and serve only this curated data through the OPTIMADE server.
```

Reply:
Ha, it was changed back after I did the rebase 😅 Done